docker-compose.yml

```diff
@@ -51,6 +52,17 @@ services:
     links:
       - web
 
+  # Google Pub/Sub emulator
+  pubsub:
+    build:
+      context: pubsub
+      dockerfile: Dockerfile
+    environment:
+      - PUBSUB_PROJECT_ID=socorro
```

Comment:
Need to add a note that this needs to match `docker/config/local_dev.env`. Also, I think it might not match now--should check that.